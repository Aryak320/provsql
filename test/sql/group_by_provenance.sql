\set ECHO none
\pset format unaligned
SET search_path TO provsql_test, provsql;

CREATE TABLE group_by_provenance_result AS
  SELECT city, sr_formula(provenance(),'personnel_name') AS formula
  FROM personnel
  GROUP BY city, provenance();

SELECT remove_provenance('group_by_provenance_result');
SELECT * FROM group_by_provenance_result ORDER BY city, formula;
DROP TABLE group_by_provenance_result;
