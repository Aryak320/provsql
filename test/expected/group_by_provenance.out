\set ECHO none
remove_provenance

city,formula
Berlin,Ellen
Berlin,Susan
New York,John
New York,Paul
Paris,Dave
Paris,Magdalen
Paris,Nancy
